.github/workflows/ci.yml

```diff
@@ -13,38 +13,35 @@ concurrency:
 
 env:
   CI: true
-  # dist: ember-headless-form/dist
+  dist: packages/ember-headless-form/dist
 
 jobs:
   install_dependencies:
     name: Install
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: TurboRepo local server
+        uses: felixmosh/turborepo-gh-artifacts@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          server-token: foo-123
```

Comment:
~I'm not seeing this in https://github.com/felixmosh/turborepo-gh-artifacts#action-inputs, but in code seeing a description of `An access token of the local turbo-server`.  Is this required? Or does it help the `Setup Tests` step in some way?~

Looks like `server-token` gets passed as `--token` to the turbo CLI (https://turbo.build/repo/docs/reference/command-line-reference#--token), eh?

Reply:
Yup. For our use case the value doesn't matter 🎉

Reply:
Last i checked, there are errors without. Like, turbo thinks setup is incomplete or something. It's been a while since i tried though, so i'll double check. Maybe something is different now.